charts/invenio/values.yaml

```diff
@@ -225,6 +225,8 @@ postgresqlExternal: {}
 
 opensearch:
   enabled: true
+  sysctlImage:
+    enabled: false
```

Comment:
I tested setting `opensearch.sysctlImage.enabled=false` now, and it seems to work. Either, the docs I found on Bitnami are outdated or the kernel that my nodes run (RHEL 9.2) happen to be compliant with the requirements of OpenSearch. Not sure which, and I don't really have time to dig deeper into it.

Not sure if this change should be part of the Invenio chart, though. If the docs are right, we can expect this to fail on some systems. Obviously it works sometimes, though. So either we decide that this is a good default value to have so we include it, or we decide that it's system dependent and should be configured by the user according to need.

What do you think? Should we keep this or remove it?

Reply:
I believe these are because of https://opensearch.org/docs/1.0/opensearch/install/important-settings/. I assume this is something that the [upstream chart addresses](https://github.com/bitnami/charts/tree/main/bitnami/opensearch#default-kernel-settings).

Reply:
Yes exactly, that's what the upstream chart tries to address. But they write that "If those values are not set in the underlying operating system, the OS containers fail to boot with ERROR messages". And I did not see any issues when starting the containers. So the question is whether the docs for `bitnami/opensearch` are out-of-date (so the images always works now) or if it was a fluke that it worked for me…

I see that you link to an old version of the OpenSearch docs. Do you mean that this is required only for older versions of OpenSearch? Or was that just a mistake?

Reply:
@slint Well, that did not last long. I just ran an installation where having `sysctlImage.enabled=false` resulted in OpenSearch crashing. I have no idea why it didn't crash for me before, perhaps it's sensitive to system load or something.

I suggest we revert the commit that introduces this particular change. **Do you agree?**

By reverting it, the chart will work for anyone that deploys in a dev environment and want OpenSearch included. And if someone (like me) wants to run it in production where a policy prohibits privileged containers, then they have to run an OpenSearch service separate from this Helm chart installation (which seems to be what you already recommend for prod).

Seems to not be related to Bitnami's packaging, but rather the architecture of [OpenSearch](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/#important-settings )and [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html), so I think there's little point trying to research this further unless you plan on dropping OpenSearch as a dependency for Invenio altogether.

Reply:
I've opened a PR to revert it here: https://github.com/inveniosoftware/helm-invenio/pull/107